adjusting_configuration.md

```diff
@@ -1,15 +1,39 @@
-When building an image on which to run containers, the Dockerfile
-adjusts configuration files by applying patches to them. The main
-cBioPortal configuration file is
-`/cbioportal/src/main/resources/portal.properties`, which is generated
-by patching `/cbioportal/src/main/resources/portal.properties` with
-`portal.properties.patch`. To build an image that uses different
-settings (see the documentation on the [main
-properties](https://github.com/cBioPortal/cbioportal/blob/master/docs/portal.properties-Reference.md)
-and the [skin properties]()), start a container, and get copies of
-`portal.properties` and `portal.properties.EXAMPLE` from the
-container:
+# Run image with changes in Dockerfile or properties
 
-```bash
+When building an image on which to run containers, the Dockerfile adjusts configuration files by applying patches to them. The main cBioPortal configuration file is `/cbioportal/src/main/resources/portal.properties`, which is generated by patching `/cbioportal/src/main/resources/portal.properties` with `portal.properties.patch`. To build an image that uses different settings (see the documentation on the [main properties](https://github.com/cBioPortal/cbioportal/blob/master/docs/portal.properties-Reference.md) and the [skin properties](https://github.com/cBioPortal/cbioportal/blob/master/docs/Customizing-your-instance-of-cBioPortal.md)), you can follow the steps listed below.
+
+#### Step 1: modify `portal.properties` or `Dockerfile` in the cBioPortal Docker directory
+
+If you want to modify the `Dockerfile`, you just have to move to the directory where you downloaded the `cbioportal-docker` repository and change it. Then, you can directly move to step 3 below. 
+
+If you want to modify `portal.properties`, you have two options:
+
+##### Option 1: if you have built a cBioPortal Docker image before
+If you have built a cBioPortal Docker image before (like the step 3 of [Readme](https://github.com/thehyve/cbioportal-docker/blob/master/README.md)), you have to retrieve the files `portal.properties` and `portal.properties.EXAMPLE` from these Docker container, be sure that you are in your cBioPortal Docker directory, and type the following command:
```

Comment:
Could you replace the comma after &lsquo;these Docker container&rsquo; (probably a typo too) by a full stop? Currently it looks like that's a step you need to figure out before you reading the rest of the sentence.